app/src/services/redisSessionStorage.js

```diff
@@ -29,7 +29,7 @@ export default class RedisSessionStorage implements SessionStorageInterface {
   set(token: string, user: User): void {
     // Sets field in the hash stored at token to user.
     // @see https://redis.io/commands/hset
-    this.client.hset("hash", token, JSON.stringify(user));
+    this.client.hset("sessions", token, JSON.stringify(user));
```

Comment:
since we're using redis to store token, I suggest to use Redis [expire](https://redis.io/commands/expire) mechanism instead of coding our own